### PR TITLE
[Docs] overview

### PR DESCRIPTION
**Pull request by plyfager**

### CODE REVIEW

File: docs/en/overview.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Overview
+
+MMGeneration is a powerful toolkit for generative models, including GANs and diffusion models. It is based on PyTorch and [MMCV](https://github.com/open-mmlab/mmcv). The master branch works with **PyTorch 1.5+**.
+
+## supported tasks
+
+Now, we support 4 tasks of image generation. Lists as follows.
```
Comment (zengyh1900) [inline docs/en/overview.md:7]: Lists are as follows.

File: docs/en/overview.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+- Unconditional GANs
+- Conditional GANs
+- Image2Image Translation
+- Internal Learning
+- Denoising Diffusion Probabilistic Models
+
+## highlight
+
+- **High-quality Training Performance:** We currently support training on Unconditional GANs, Internal GANs, and Image Translation Models. Support for conditional models will come soon.
```
Comment (zengyh1900) [inline docs/en/overview.md:17]: Do you miss the 'diffusion model'? 
What does 'conditional models' mean?

Comment (zengyh1900) [reply docs/en/overview.md:17]: Using 'MMGeneration' instead of 'we' would be better.

File: docs/en/overview.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- **Powerful Application Toolkit:** A plentiful toolkit containing multiple applications in GANs is provided to users. GAN interpolation, GAN projection, and GAN manipulations are integrated into our framework. It's time to play with your GANs! ([Tutorial for applications](docs/en/tutorials/applications.md))
```
Comment (zengyh1900) [inline docs/en/overview.md:18]: 'A plentiful toolkit containing multiple applications in GANs is provided to users.'
--->
'A toolkit that provides plentiful applications to users.'

'GAN interpolation, GAN projection, and GAN manipulations are integrated into our framework.'
---> 
'MMGeneration supports GAN interpolation, GAN projection, GAN manipulations and many other popular GAN's applications'

File: docs/en/overview.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+To get started with our repo, please refer to [get_started.md](docs/en/get_started.md).
+
+## user guides
+
+For elementary guides on basic usage, please refer to [user_guides](docs/en/user_guides).
+
+## advanced guides
+
+To learn design and structure of our framework, as well as how to extend the repo, how to use multiple repos and other advanced usages, please refer to [advanced_guides](docs/en/advanced_guides).
```
Comment (zengyh1900) [inline docs/en/overview.md:30]: use 'MMGeneration', instead of 'our framework'

File: docs/en/overview.md (hunk 3 above)
Comment (Z-Fran) [inline docs/en/overview.md:18]: Link is unavailable. Should be (/docs/en/tutorials/applications.md)

File: docs/en/overview.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- Denoising Diffusion Probabilistic Models
+
+## highlight
+
+- **High-quality Training Performance:** We currently support training on Unconditional GANs, Internal GANs, and Image Translation Models. Support for conditional models will come soon.
+- **Powerful Application Toolkit:** A plentiful toolkit containing multiple applications in GANs is provided to users. GAN interpolation, GAN projection, and GAN manipulations are integrated into our framework. It's time to play with your GANs! ([Tutorial for applications](docs/en/tutorials/applications.md))
+
+## get started
+
+To get started with our repo, please refer to [get_started.md](docs/en/get_started.md).
```
Comment (Z-Fran) [inline docs/en/overview.md:22]: Link is unavailable. May be (/docs/en/get_started.md)

File: docs/en/overview.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## user guides
+
+For elementary guides on basic usage, please refer to [user_guides](docs/en/user_guides).
```
Comment (Z-Fran) [inline docs/en/overview.md:26]: Link is unavailable. May be (/docs/en/user_guides)

File: docs/en/overview.md (hunk 4 above)
Comment (Z-Fran) [inline docs/en/overview.md:30]: Link is unavailable. May be (/docs/en/advanced_guides)

File: docs/en/overview.md (hunk 5 above)
Comment (plyfager) [reply docs/en/overview.md:22]: in fact, we are supposed to use relative path. ie get_started.md

File: docs/en/overview.md (hunk 3 above)
Comment (plyfager) [reply docs/en/overview.md:18]: I'll add a extra ``applications.md``.

**Pull request merged**